bugbug/bug_features.py

```diff
@@ -134,6 +134,10 @@ def cleanup_fileref(text):
     return re.sub(r'\w+\.py\b|\w+\.json\b|\w+\.js\b|\w+\.jsm\b|\w+\.html\b|\w+\.css\b|\w+\.c\b|\w+\.cpp\b|\w+\.h\b', '__FILE_REFERENCE__', text)
 
 
+def cleanup_hex(text):
+    return re.sub(r'[^\s]*0[xX][0-9a-fA-F]+', '__HEX_NUMBER__', text)
```

Comment:
Same here, you can use `\b` to define a word boundary (and it should be both before and after the number).